### DISCUSSION TITLE
今天月亮好美，一起赏月吗？

### DISCUSSION THREAD

**Discussion by RETRN1** [share121/inter-knot#1677] [其他]
![20240812153108_edit_124299679356175](https://github.com/user-attachments/assets/57cb2470-b801-4867-a802-94042a31645d)
从未见过如此美丽的月亮，就是怎么黑了一半...空洞跑月亮上了？😱😱😱

**Comment by gaojiaobei** [comment]
害怕😰

**Comment by lllliufeng** [comment]
猜一手月亮上的以骸有多大
我猜有一个新艾莉都那么大（
